### ISSUE TITLE
Snap to grid ? Resizeable ?

### ISSUE THREAD

**Issue by daslicht** [xieziyu/angular2-draggable#35]
Hi , 
is it possible to have teh same featuites as jQuery draggable, please ? such as:

- snap to grid
- resizeable

?

**Comment by fabich** [comment]
@xieziyu are there any plans to integrate this feature? (snap to grid)

I guess this is the same as #66?

**Comment by xieziyu** [comment]
@fabich 
Please use [gridSize] option to do snapping.

**Comment by daslicht** [comment]
https://xieziyu.github.io/angular2-draggable/#/advance/snap-grid

So now only resizeable is missing?

**Comment by xieziyu** [comment]
@fabich Yes, I had. But I'm up to my neck in work recently. Sorry about that.

**Comment by fabich** [comment]
no worries :)
thanks for providing this lib 👍

**Comment by xieziyu** [comment]
@fabich @daslicht 
I've published `2.0.0-beta.0` version to support `ngResizable` directive.
